README.md

```diff
@@ -151,9 +151,9 @@ feet, in a certain section, is to not use `await` in it.
 
 ### Error handling
 
-Exceptions inheriting from `CatchableError` are caught by hidden `try` blocks
-and placed in the `Future.error` field, changing the future's status to
-`Failed`.
+Exceptions inheriting from `Exception` (or `CatchableError` when
```

Comment:
We should never catch non-`CatchableError` really - in particular, we must not catch `Defect` at all